pkg/datapath/iptables/iptables.go

```diff
@@ -1794,3 +1800,65 @@ func (m *Manager) addCiliumENIRules() error {
 		"-m", "comment", "--comment", "cilium: primary ENI",
 		"-j", "CONNMARK", "--restore-mark", "--nfmask", nfmask, "--ctmask", ctmask})
 }
+
+func (m *Manager) addAWSCNIRPFilterRules() error {
+	if !m.sharedCfg.EnableIPv4 {
+		return nil
+	}
+	ctMarkIsProxy := fmt.Sprintf("%#08x", linux_defaults.CTMarkIsProxy)
+
+	// Track connections which originate from a remote peer and terminate at an
+	// ingress proxy. This traffic will ingress into the node on vlan.eth.+,
+	// replies from the proxy will egress out of the node on vlan.eth.+, this
+	// would still fail rpfilter however as the reverse FIB lookup can't
```

Comment:
Without understanding the whole context, should we also be manipulating the rp_filter sysctl?

Personally, I'm not so sure; as a chained plugin we don't necessarily "own" the interface sysctls. However, if we need to change a sysctl to make it work, then we should do that :-).

Reply:
Yes, I really didn't want to have to touch these settings.